modules/library/coverage/src/main/java/org/geotools/resources/coverage/FeatureUtilities.java

```diff
@@ -371,4 +382,32 @@ public static double area(Point2D[] polyPoints) {
 		area /= 2.0;
 		return (area);
 	}
+
+    public static Properties loadPropertiesFromURL(URL propsURL) {
```

Comment:
My fault. I put that method here before moving it to CoverageUtilities.
Going to remove it